python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -159,3 +159,17 @@ def test_reset_index_multiindex():
         .reset_index()
         .merge(ddf_lookup, on="id_1"),
     )
+
+
+@pytest.mark.parametrize("split_out", [1, 2, 3])
+def test_groupby_split_out(split_out):
```

Comment:
Could we add a test when grouping by more than one column, and when grouping by a string column and see if things break.

Reply:
Sounds good to me - Grouping by a string column does cause problems (looking into it now)